src/libraries/Microsoft.Extensions.Logging.Console/src/SimpleConsoleFormatter.cs

```diff
@@ -147,7 +147,13 @@ private static string GetLogLevelString(LogLevel logLevel)
 
         private ConsoleColors GetLogLevelConsoleColors(LogLevel logLevel)
         {
-            if (FormatterOptions.DisableColors)
+            bool disableColors = FormatterOptions.ColorBehavior switch
```

Comment:
>ormatterOptions.ColorBehavior switch [](start = 34, length = 36)

nit: would be better to convert this to simple initialization than using the switch statement I guess of the sake of readability:

disableColors = (FormatterOptions.ColorBehavior == LoggerColorBehavior.Disabled) || (FormatterOptions.ColorBehavior == LoggerColorBehavior.Default &&  System.Console.IsOutputRedirected);